src/inspector/models/notation/lines/textlinesettingsmodel.cpp

```diff
@@ -296,6 +296,15 @@ void TextLineSettingsModel::onUpdateLinePropertiesAvailability()
 
     m_dashLineLength->setIsEnabled(isLineAvailable && areDashPropertiesAvailable);
     m_dashGapLength->setIsEnabled(isLineAvailable && areDashPropertiesAvailable);
+
+    bool hasBeginText = !m_beginningText->value().toString().isEmpty();
+    bool hasContinueText = !m_continuousText->value().toString().isEmpty();
+    bool hasEndText = !m_endText->value().toString().isEmpty();
+
+    m_beginningTextOffset->setIsEnabled(hasBeginText);
+    m_continuousTextOffset->setIsEnabled(hasContinueText);
+    m_endTextOffset->setIsEnabled(hasEndText);
+    m_gapBetweenTextAndLine->setIsEnabled(isLineAvailable && (hasBeginText || hasContinueText));
```

Comment:
Now that #22602 has been merged: 
```suggestion
    m_gapBetweenTextAndLine->setIsEnabled(isLineAvailable && (hasBeginText || hasContinueText || hasEndText));
```

Reply:
I had forwarded the question above to @bkunda on Slack, but unless I missed something I hadn't heard back yet. I think it's indeed good to move this property but the question is of course where.

Reply:
Sorry for the delay, but we have an answer now: let's move it to the bottom of the text section, below the "end text" group. Thanks in advance!

Reply:
No problem! Thanks for the taking the time